### PR TITLE
Add `servicetalk-utils-internal` module

### LINKED ISSUE #189: Consolidate duplicate MathUtil
http-api and opentracing both have a duplicate `MathUtil` file. We should share code between these two modules via an `internal` module.

### PR DESCRIPTION
**Pull request by heowc**
__Motivation__

We need to `servicetalk-utils-internal` module

__Modifications__

Remove duplicate class(`MathUtil`)

__Result__

Resolves #189

### CODE REVIEW

File: servicetalk-utils-internal/src/main/java/io/servicetalk/utils/internal/MathUtil.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2018 Apple Inc. and the ServiceTalk project authors
+ * Copyright © 2019 Apple Inc. and the ServiceTalk project authors
```
Comment (idelpivnitskiy) [inline servicetalk-utils-internal/src/main/java/io/servicetalk/utils/internal/MathUtil.java]: Please, change the year to `2018-2019`

File: servicetalk-utils-internal/build.gradle [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+    implementation project(":servicetalk-annotations")
+    implementation "com.google.code.findbugs:jsr305:$jsr305Version"
+    implementation "org.slf4j:slf4j-api:$slf4jVersion"
```
Comment (idelpivnitskiy) [inline servicetalk-utils-internal/build.gradle]: This module does not use `slf4j`, please remove this dependency.

**Comment by NiteshKant** [comment]
@servicetalk-bot test this please

**Comment by heowc** [comment]
Thank for the review!
I fixed it :laughing:

**Comment by NiteshKant** [comment]
@servicetalk-bot test this please

**Pull request merged**